cancer_ai/rewards.py

```diff
@@ -0,0 +1,56 @@
+   
+class Rewarder():
+    def __init__(self, config=None):
+        # competition_id to (hotkey_uid, days_as_leader)
```

Comment:
you shouldn't use hotkey_uid, there can be hotkey order rotation, use hotkey

Reply:
Ok then I would expect hotkey itself not hotkey_id as an output of the evaluation from your site right?

Reply:
that's right.